tgui/packages/tgui/interfaces/LobbyMenu.tsx

```diff
@@ -0,0 +1,608 @@
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+                <Stack.Item>
+                  <Stack justify="center">
+                    <Stack.Item>
+                      <Box
+                        className="typeEffect"
+                        style={{
+                          animationDelay: '1.4s',
+                        }}
+                      >
+                        {character_name}
+                      </Box>
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+                <Stack.Item>
+                  <Stack justify="center">
+                    <Stack.Item>
+                      <Box
+                        className="typeEffect hiveEffect"
+                        style={{
+                          animationDelay: '1.4s',
+                        }}
+                      >
+                        {xenoName}
+                      </Box>
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+          </Stack>
+        </Stack.Item>
+
+        <TimedDivider />
+
+        <LobbyButton
+          index={1}
+          onClick={() => act('tutorial')}
+          disabled={!tutorials_ready}
+          icon="book-open"
+        >
+          Tutorial
+        </LobbyButton>
+        <LobbyButton
+          index={2}
+          onClick={() => act('preferences')}
+          icon="file-lines"
+        >
+          Setup Character
+        </LobbyButton>
+        <LobbyButton index={3} onClick={() => act('playtimes')} icon="list-ul">
+          View Playtimes
+        </LobbyButton>
+
+        <TimedDivider />
+
+        <LobbyButton
+          index={4}
+          icon="eye"
+          onClick={() => {
+            setModal(
+              <ModalConfirm>
+                <Box>
+                  <Stack vertical>
+                    <Stack.Item>Are you sure you wish to observe?</Stack.Item>
+                    <Stack.Item>
+                      When you observe, you will not be able to join as marine.
+                    </Stack.Item>
+                    <Stack.Item>
+                      It might also take some time to become a xeno or
+                      responder!
+                    </Stack.Item>
+                  </Stack>
+                  <Stack justify="center">
+                    <Stack.Item>
+                      <Button onClick={() => act('observe')}>Confirm</Button>
+                    </Stack.Item>
+                  </Stack>
+                </Box>
+              </ModalConfirm>,
+            );
+          }}
+        >
+          Observe
+        </LobbyButton>
+
+        {round_start ? (
+          <Stack.Item>
+            <Stack>
+              <Stack.Item grow>
+                <LobbyButton
+                  index={5}
+                  disabled={!!readied}
```

Comment:
readied should be a boolean value so this should be redundant
If it has a chance of being undefined, you can declare a default by just doing something like
```tsx
  const {
    // ...
    readied = false,
    // ...
  } = data;

Reply:
ah wait, this might be a byond boolean (1,0), so this is probably fine